static/js/event/event.ts

```diff
@@ -18,14 +18,49 @@
  * Disaster events
  */
 
+import axios from "axios";
 import React from "react";
 import ReactDOM from "react-dom";
 
-import { Page } from "./page";
+import { EventPage, Page } from "./page";
+import { Property } from "./types";
+
+function parseApiResponse(resp): Array<Property> {
+  // Parses response from triples API for later rendering
+  const parsedResponse = [];
+  for (const [property, connectedNodes] of Object.entries(resp)) {
+    for (const nodes of Object.values(connectedNodes)) {
+      parsedResponse.push({ name: property, values: nodes });
```

Comment:
I'm curious what the intended behavior here is. For parsedResponse, did you want:
a. one list item per property
b. one list item per list of nodes in connectedNodes

I think currently you are doing (b).

Reply:
Intended behavior is (b). Updated the logic to make it more clear what's happening.